plugins/utils/todo-comments.nix

```diff
@@ -398,11 +397,11 @@ lib.nixvim.neovim-plugin.mkNeovimPlugin config {
               default = null;
               example = "TODO,FIX";
             };
-
-            options = keymaps.mapConfigOptions;
           };
-      in
-      mapAttrs mkKeymapOption keymapsActions;
+        };
+        description = "Keymap for function ${funcName}";
+      }
+    ) keymapsActions;
```

Comment:
I think we can inline `keymapsActions` now, since it'll only be used here.

I'd consider using a list together with `lib.nixvim.upperFirstChar` instead of an attrset, though this isn't too important.